src/validators/v1/organization-extensions.js

```diff
@@ -35,6 +42,30 @@ module.exports = {
 			.isIn(allowedResourceTypes)
 			.withMessage(`resource_type is invalid, must be one of: ${allowedResourceTypes.join(', ')}`)
 
+		req.checkBody('data_managers')
+			.optional({ checkFalsy: true })
+			.custom((value) => {
+				// Allow empty array explicitly
+				if (Array.isArray(value) && value.length === 0) {
```

Comment:
@adithyadinesh0412  why do we need to add data_manager as empty array, if they dont want to add they can skip this field

Reply:
@priyanka-TL , Only for update we are allowing to pass data_managers as empty array. Just in case they have to remove the roles we don't have any other option otherthan making the data_managers empty.